styles/index.css

```diff
@@ -12,3 +12,7 @@
 @import './base.css';
 @import './markdown.css';
 @import './effects.css';
+
+html {
```

Comment:
Could you set this within the base.css file? We already have a HTML tag there 👀

Reply:
Yeah , I'll update this wait, when adding this to directly to base.css require the server to cold start to see the changes but it work with hot reloading when I added it to directly in index.css. But it doesnt affect the prod. wait I'm updating the file again.